packages/react/src/portal/Portal.tsx

```diff
@@ -0,0 +1,63 @@
+'use client';
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { FloatingPortal } from '@floating-ui/react';
+import { usePortalContext } from './PortalContext';
+import { HTMLElementType, refType } from '../utils/proptypes';
+
+/**
+ * Renders a portal element into the DOM.
+ *
+ * Documentation: [Base UI Portal](https://base-ui.com/react/components/portal)
+ */
```

Comment:
This doesn't really exist (and probably shouldn't). Can't do `@ignore - internal component.` either